doc/HttpLuaModule.wiki

```diff
@@ -1147,6 +1147,36 @@ Similar to [[#init_worker_by_lua|init_worker_by_lua]], but accepts the file path
 
 This directive was first introduced in the <code>v0.9.5</code> release.
 
+== exit_worker_by_lua_block ==
+
+'''syntax:''' ''exit_worker_by_lua_block { lua-script }''
+
+'''context:''' ''http''
+
+'''phase:''' ''exiting-worker''
+
+Runs the specified Lua code upon every Nginx worker process's exiting when the master process is enabled.
+
+This hook is often used to release resources when init_worker_by* create resources, To prevent the worker exit abnormal.
```

Comment:
@agentzh I add init_worker_by_lua* label to init_worker_by_lua_block not init_worker_by_lua.  I want recommend others use init_worker_by_lua_block. It has a problem?